### DISCUSSION TITLE
Do you recommend minifying the class names?

### DISCUSSION THREAD

**Discussion by gfpacheco** [tailwindlabs/tailwindcss#1557] [General]
Since this is already a utility based css library, would you recommend using a tool to rename the classes in an attempt to minify the generated css even more? Something like https://github.com/google/postcss-rename

**Comment by cossssmin** [comment]
I think it's more trouble than it's worth - you'd also need to rename the HTML. And if you're using PurgeCSS, you'd need to do it after that, otherwise you'd be left with no CSS.

**Comment by kirkbushell** [comment]
Tbh, it's not a bad idea - considering how much of our styles are now also in our html, being able to rename the class names and the HTML would be quite beneficial.

I suspect you'd be able to do this with a tailwind plugin that runs at build time.
